jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -1184,6 +1197,15 @@ def _load_regions(self, regions, edit_subset=None, combination_mode=None, max_nu
 
         if not isinstance(regions, (list, tuple, Regions, SpectralRegion)):
             regions = [regions]
+        if isinstance(subset_label, str):
+            subset_label = [subset_label]
+
+        bad_labels = []
+        for label in subset_label:
+            if not self.app._check_valid_subset_label(label, raise_if_invalid=False):
+                bad_labels.append(label)
+        if len(bad_labels > 0):
```

Comment:
should be `len(bad_labels) > 0`

Reply:
Oh what the heck, good catch. I guess I broke this and then didn't test after, I'll fix it shortly.

Reply:
Fixed.